app/fissile.go

```diff
@@ -771,12 +771,54 @@ func (f *Fissile) GenerateKube(rolesManifestPath, outputDir, repository, registr
 		return err
 	}
 
+	// Phase I (a): Gather the secrets
+
+	cvs := model.MakeMapOfVariables(rolesManifest)
+	for key, value := range cvs {
+		if !value.Secret {
+			delete(cvs, key)
+		}
+	}
+	// cvs now holds only the secrets.
+	thesecrets, therefs, err := kube.MakeSecrets(cvs, defaults)
```

Comment:
I don't think we need `the` on the variable names (`secrets`, `refs` are perfectly fine by themselves).